core/embed/upymod/modtrezorutils/modtrezorutils.c

```diff
@@ -325,7 +328,7 @@ STATIC mp_obj_t mod_trezorutils_check_free_heap(mp_obj_t arg) {
   if (free_heap > info.free) {
     gc_dump_info();
     mp_raise_msg_varg(&mp_type_AssertionError,
-                      "Free heap decreased by " UINT_FMT " bytes",
+                      MP_ERROR_TEXT("Free heap size decreased"),
```

Comment:
As `UINT_FMT` was removed, the argument should be too.

Reply:
You're right - fixed in https://github.com/trezor/trezor-firmware/pull/4961/commits/d26fb6a9726cb43f064580b4d1723767ebf1d258.